lib/ui/viewer/gallery/gallery_overlay_widget.dart

```diff
@@ -5,7 +5,6 @@ import 'dart:ui';
 import 'package:flutter/cupertino.dart';
```

Comment:
This file can be deleted once we use the new selection bottom overlay on trash screen. Will do it in the next PR.